src/assets/Doneth.sol.txt

```diff
@@ -58,8 +55,38 @@ contract Doneth {
         addMember(msg.sender, 1, true, _founderName);
     }
 
+    mapping(address => ERC20) public tokens;
+    address[] public tokenKeys;
+    struct Token {
+        bool exists;
+        uint256 totalWithdrawn;
+    }
+
+    // deposit to account
+    // means that user sends ERC20 to ERC20 address with contract address as owner
+    // balance is kept on corresponding contract
+
+    function checkERC20Balance(address token) public returns(uint256) {
+        return ERC20(token).balanceOf(address(this));
+    }
+
+    function withdrawToken(uint256 amount, address token) public onlyExisting(msg.sender) {
+        uint256 newTotal = calculateTotalWithdrawableTokenAmount(msg.sender, token);
```

Comment:
Is this function implemented anywhere?

Reply:
this function is incomplete but would be the method similar to a normal withdraw where a member could withdraw their token amount